### PR TITLE
Rwd helpers react

### LINKED ISSUE #1043: JSX: helpers

### PR DESCRIPTION
**Pull request by matikrk**
* component
* examples
* tests

Closes #1043

### CODE REVIEW

File: src/components/helpers/RwdHelper.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const types = {
```
Comment (aju) [inline src/components/helpers/RwdHelper.jsx]: lets already use TYPE and SMALL_ONLY

File: src/components/helpers/RwdHelper.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  const finalClassName = classNames(children.props.className, hideClass);
+
+  return React.cloneElement(children, {className: finalClassName});
+};
+
+RwdHelper.propTypes = {
+  // One child only !!!
+  children: PropTypes.oneOfType([PropTypes.element, PropTypes.string]).isRequired,
```
Comment (aju) [inline src/components/helpers/RwdHelper.jsx]: why only one child ?

File: src/components/helpers/pages/rwd.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+              </span>
+              - hidden for small and medium screens
+            </span>
+          </RwdHelper>
+        </li>
+
+        <li>
+          <RwdHelper hide={types.mediumUp}>
+            <span>
+              <span>
```
Comment (aju) [inline src/components/helpers/pages/rwd.jsx]: why span in span ?

Comment (matikrk) [inline src/components/helpers/pages/rwd.jsx]: All react components must have root and RwdHelper is kind of decorator with give className only.

File: src/components/helpers/RwdHelper.jsx (hunk 1 above)
Comment (matikrk) [inline src/components/helpers/RwdHelper.jsx]: done

**Pull request merged**